react/components/StorePickupItem.tsx

```diff
@@ -22,7 +22,7 @@ const StorePickupItem: FC<Props> = ({ store }) => {
     <div className={`flex flex-column t-body lh-copy c-muted-2 ${handles.pickupItem}`}>
       <span className={`t-heading-6 c-on-base ${handles.pickupName}`}>{`${friendlyName}`}</span>
       <span className={handles.pickupAddress}>{`${address.street}${address.number ? `, ${address.number}` : ''}`}</span>
-      <div className={`${shippingEstimate ? handles.pickupEstimate : handles.pickupUnavailable} flex`}>
+      <div className={`${shippingEstimate ? `${handles.pickupEstimate} ${handles.pickupEstimate}-${shippingEstimate}` : handles.pickupUnavailable} flex`}>
```

Comment:
```suggestion
      <div className={`${shippingEstimate ? `${handles.pickupEstimate} ${handles.pickupEstimate}--${shippingEstimate}` : handles.pickupUnavailable} flex`}>
```

this seems to me like a modifier, and the convention is to use `--` in that case